shared/misc/upload_ansible_roles_to_galaxy.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python2
+
+from __future__ import print_function
+
+from tempfile import mkdtemp
+import os
+import os.path
+import sys
+import shutil
+import re
+import argparse
+import getpass
+import yaml
+
+try:
+    from github import Github, InputGitAuthor
+except ImportError:
+    sys.stderr.write("Please install PyGithub, on Fedora it's in the "
+                     "python-PyGithub package.\n")
+    sys.exit(1)
+
+
+ORGANIZATION_NAME = "Ansible-Security-Compliance"
+GIT_COMMIT_AUTHOR_NAME = "SCAP Security Guide development team"
+GIT_COMMIT_AUTHOR_EMAIL = "scap-security-guide@lists.fedorahosted.org"
+META_TEMPLATE_PATH = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)),
+    "ansible_galaxy_meta_template.yml"
+)
+README_TEMPLATE_PATH = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)),
+    "ansible_galaxy_readme_template.md"
+)
+
+
+def create_empty_repositories(github_new_repos, github_org):
+    for github_new_repo in github_new_repos:
+        print("Creating new Github repository: %s" % github_new_repo)
+        github_org.create_repo(
+            github_new_repo,
+            description="Role generated from SCAP Security Guide",
+            homepage="https://www.open-scap.org/",
+            private=False,
+            has_issues=False,
+            has_wiki=False,
+            has_downloads=False)
+
+
+def clone_and_init_repository(parent_dir, repo):
+    os.system(
+        "git clone git@github.com:%s/%s.git" % (ORGANIZATION_NAME, repo))
+    os.system("ansible-galaxy init " + repo + " --force")
+    os.chdir(repo)
+    try:
+        os.system('git add .')
+        os.system('git commit -a -m "Initial commit" --author "%s <%s>"'
```

Comment:
I don't like much this initial commit message. Can we maybe add some info from which SSG release or commit this Playbook was built?

Reply:
The initial commit really does nothing besides commit the initial ansible galaxy template with no changes. However this is a good point, I will change description of repository to say which version of ssg it was genertaed from. Perhaps even the readme

Reply:
Turns out I can't get that info, at least not easily. This will have to wait for future improvements I think.